lib/db_utils.pm

```diff
@@ -67,15 +67,20 @@ sub influxdb_push_data {
     my ($url, $db, $org, $token, $data, %args) = @_;
     $args{quiet} //= 1;
     $args{proceed_on_failure} //= 0;
+    $args{timeout} //= 90;
     $data = build_influx_query($data);
+    # connectivity pre-check
+    if (script_run('curl --silent --insecure --fail ' . $url . '>/dev/null', timeout => $args{timeout}, die_on_timeout => 0)) {    # curl error
```

Comment:
```suggestion
    if (script_run('curl --silent --insecure --fail ' . $url . '>/dev/null', timeout => $args{timeout}, die_on_timeout => 1)) {    # curl error
```

I think a timeout is to be treated as "database not available", no?

Reply:
No: the aim of this PR is to check the status and on erroro to skip only to next step: adding that `1` in timeout case the test will crash.

Reply:
And as I said - this "crash" or timeout means that the database is not available and should be treated as such, no?

Reply:
No, that performance database should not impact openqa P.C. tests in case of any of its eventual issue, as agreed with @asmorodskyi.

Reply:
this is BTW insertion of stat into InfluxDB which we doing during ALL instance creation and it should not affect each and every test execution . It is correct to mute exception in this case

Reply:
AFAICS this fails, if the database reports an error but will not fail, if the connection times out. That doesn't make sense, a timeout is for this check IMHO the same as if the database would be down.

If this check runs into a timeout, the push later on will also run into issues.

Reply:
this is why I think we should simply wrap everything into `eval` but still inside this eval it would make sense to do fast track check if DB is available . this check should be with short timeout

Reply:
Ah, I see now what you mean. You're right, this is fine as it is.